src/engine/SCons/Environment.xml

```diff
@@ -844,7 +844,7 @@ env3 = env.Clone(CCFLAGS = '-g')
 
 <para>
 Additionally, a list of tools and a toolpath may be specified, as in
-the Environment constructor:
+the &f-link-Environment; constructor:
```

Comment:
Does this create hyperlink?

Reply:
Yes. The doc build constructs all those in the generated files for the definitions in a `<builder>`, `<function>`, etc.  This one already exists in the checked-in generated docs:
```
doc/generated/functions.mod:<!ENTITY f-link-Environment "<link linkend='f-Environment' xmlns='http://www.scons.org/dbxsd/v1.0'><function>Environment</function></link>">
```